src/main/kotlin/xyz/crossward/repository/EntryRepository.kt

```diff
@@ -4,15 +4,27 @@ import org.springframework.data.jpa.repository.Query
 import org.springframework.data.repository.CrudRepository
 import org.springframework.stereotype.Repository
 import xyz.crossward.entities.Entry
+import xyz.crossward.entities.EntryId
 import java.time.LocalDate
 import java.util.stream.Stream
 
 @Repository
-interface EntryRepository : CrudRepository<Entry, String> {
+interface EntryRepository : CrudRepository<Entry, EntryId> {
 
     @Query("select count(e) from Entry e where e.userId = :userId and e.date = :date")
     fun count(userId: String, date: LocalDate?): Long
 
     @Query("select e.time from Entry e where e.userId = :userId")
-    fun findTimesByUserId(userId: String): Stream<Int>
+    fun getTimesByUserId(userId: String): Stream<Int>
+
+    @Query(
+        """
+        select e from Entry e
```

Comment:
Could we say 
```suggestion
        select e.time, e.date from Entry e
```

so we don't need the redundant userId in the response

Reply:
I'll have to create a new Entry response object because the userId can't be null, but I can definitely make that change if you want.

Reply:
I think it would be worth it, probably wouldn't save us _that_ much data but definitely some